### PR TITLE
Tweak repo header

### PR DESCRIPTION
**Pull request by silverwind**
- Tweak colors, remove link color from repo name and make text use inherited color
- Downsize repo icon from 32px to 24px

Before:
Screenshot 2024-02-11 at 15 31 00

After:
Screenshot 2024-02-11 at 15 30 48

### CODE REVIEW

**Review by delvh — APPROVED** [review]

**Comment by techknowlogick** [comment]
I know GH doesn't emphasize that the repo name is a link, but I feel like this downplays an important aspect of the page. I guess this isn't exactly the title/h1 of the page though.

That being said, these are feelings that aren't backed by any usability data, so I won't block. Just wanted to register my thoughts.

**Comment by denyskon** [comment]
I don't want for the page to become more and more greyscale 😅

I like the primary color as a way to highlight links, but I won't block if it gets enough approvals.

File: templates/repo/header.tmpl [hunk 1]
```diff
@@ -6,8 +6,8 @@
 				<div class="flex-item-leading">{{template "repo/icon" .}}</div>
 				<div class="flex-item-main">
 					<div class="flex-item-title">
-						<a class="text light thin" href="{{.Owner.HomeLink}}">{{.Owner.Name}}</a>/
-						<a class="text primary" href="{{$.RepoLink}}">{{.Name}}</a></div>
+						<a class="text muted thin" href="{{.Owner.HomeLink}}">{{.Owner.Name}}</a>/
```
Comment (wxiaoguang) [inline templates/repo/header.tmpl]:
```suggestion
						<a class="muted gt-font-normal" href="{{.Owner.HomeLink}}">{{.Owner.Name}}</a>/
```

File: templates/repo/header.tmpl [hunk 2]
```diff
@@ -6,8 +6,8 @@
...
+						<a class="text muted" href="{{$.RepoLink}}">{{.Name}}</a></div>
```
Comment (wxiaoguang) [inline templates/repo/header.tmpl]:
```suggestion
						<a class="muted" href="{{$.RepoLink}}">{{.Name}}</a></div>
```

**Comment by silverwind** [comment]
I think I prefer either both user and repo colored (like in 1.21), or none (like here). I do think we can increase the font size on the header a bit more to emphasize it. It was 21px on 1.21, now is 16px. Maybe 20px to match github's repo frontpage. In the navbar, they have 14px.

**Comment by silverwind** [comment]
Here's with 18px now. 20px like GitHub looked too large imho.

image

Could revert back to color, but I don't really like it, it does look "cleaner" without.

image

**Comment by techknowlogick** [comment]
@silverwind thanks for adjusting. The new screenshot without colour is a +1 from me :)

**Review by wxiaoguang — APPROVED** [review]
I agree with that it doesn't need to be that colorful.

**Comment by silverwind** [comment]
Merging as there are no strong objections.

**Pull request merged**